control-plane/roles/monitoring/templates/grafana_dashboards/s3-cluster.yaml

```diff
@@ -0,0 +1,2686 @@
+---
```

Comment:
Not necessary, component is not part of the metal-stack. Can be cleaned up.

Reply:
https://github.com/metal-stack/metal-roles/pull/115/commits/bdb0827a90f520d73aa292b008d6ac00dfa8a182